code/datums/mutations/chameleon.dm

```diff
@@ -7,26 +7,26 @@
 	text_gain_indication = "<span class='notice'>You feel one with your surroundings.</span>"
 	text_lose_indication = "<span class='notice'>You feel oddly exposed.</span>"
 	time_coeff = 5
-	instability = 35
+	instability = 35 //SKYRAT EDIT BEGIN
 	power = /obj/effect/proc_holder/spell/self/chameleon_skin_activate
 
 /obj/effect/proc_holder/spell/self/chameleon_skin_activate
 	name = "Activate Chameleon Skin"
 	desc = "The chromatophores in your skin adjust to your surroundings, as long as you stay still."
 	clothes_req = FALSE
 	action_icon = 'icons/mob/actions/actions_minor_antag.dmi'
-	action_icon_state = "ninja_cloak"
+	action_icon_state = "ninja_cloak" //SKYRAT EDIT END
 
 /datum/mutation/human/chameleon/on_acquiring(mob/living/carbon/human/owner)
 	if(..())
 		return
-	if(HAS_TRAIT(owner,TRAIT_CHAMELEON_SKIN))
-		return
+	if(HAS_TRAIT(owner,TRAIT_CHAMELEON_SKIN)) //SKYRAT EDIT BEGIN
+		return //SKYRAT EDIT END
 	owner.alpha = CHAMELEON_MUTATION_DEFAULT_TRANSPARENCY
 	RegisterSignal(owner, COMSIG_MOVABLE_MOVED, .proc/on_move)
 	RegisterSignal(owner, COMSIG_HUMAN_EARLY_UNARMED_ATTACK, .proc/on_attack_hand)
 
-/obj/effect/proc_holder/spell/self/chameleon_skin_activate/cast(list/targets, mob/user = usr)
+/obj/effect/proc_holder/spell/self/chameleon_skin_activate/cast(list/targets, mob/user = usr) //SKYRAT EDIT BEGIN
```

Comment:
If those two procs are self-contained, then they should be moved to modular. Man, I really wasn't awake when I reviewed that last PR, my apologies.